### DISCUSSION TITLE
Import / Export all routes

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#355] [Q&A] [answered]
B14:
I exported my routes via export ALL into json file, but on the other devices when importing only the first route group is imported, not ALL.

**Comment by devemux86** [comment]
"Export all" routes should create a zip files with group folders inside?
Can you see the routes inside the zip's group folders?

Can you share the zip file to check its structure?
(can send email)

**Reply by DolomitesTom** [reply]
Yes, a zip file is created. When I export each individual group by name then it works, but when I use export ALL, then only the first group is in the zip file. I guess you will simulate it easily.

**Reply by devemux86** [reply]
I see what happens. It does not export the new changes in the groups / routes.
If you save the routes (press OK) and then reopen, then "Export all" should work.

I will fix it and it will work properly in the next Cruiser update.

**Comment by DolomitesTom** [comment]
another issue:
in B14 the BRouter offline options are not complete. I am missing the menu entries others and extended or they are not visible because of a bigger font size on the other device, but I can't scroll down.

**Reply by devemux86** [reply]
@DolomitesTom
BRouter "Other" + "Advanced" options appear when "Extended settings" is enabled.

**Reply by DolomitesTom** [reply]
ah ok thanks

**Comment by DolomitesTom** [comment]
one little problem is also, when I export a routing group by name, then the proposed file name is not the name of the group, but it is date/time again. Can you name the exported file same as the group?

**Reply by devemux86** [reply]
- Press the menu of the group
- Select "Export"
- In the export options window press the name of the group
- Select export format
- Android file browser shows groupname.zip

Do you follow these steps?

**Reply by DolomitesTom** [reply]
OK works now thx

**Comment by devemux86** [comment, marked as answer]
Fixed in [Cruiser 3.0.34](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
